examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,22 +21,34 @@
      "name": "stdout",
```

Comment:
<p>I'd use kind="hist" here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
um this comes out bad:
<img width="551" alt="image" src="https://user-images.githubusercontent.com/55153708/113474383-56a6be80-948d-11eb-80fe-70f382d677ef.png">